src/bepasty/tests/test_rest_server.py

```diff
@@ -14,8 +14,8 @@
 
 import pytest
 
-from ..app import create_app
-from ..config import Config
+from bepasty.app import create_app
+from bepasty import config
```

Comment:
please keep the relative imports.

Reply:
the 2nd one would then be `from .. import config` I guess.

Reply:
I find the relative imports look a bit out of place, but I reverted anyways.